cfme/services/dashboard/ssui.py

```diff
@@ -38,7 +40,9 @@ class MyServiceForm(SSUIBaseLoggedInPage):
 
 class MyServicesView(MyServiceForm):
     title = Text(locator='//li[@class="active"]')
-    results = Text(locator='//div[contains(@class, "toolbar-pf-results")]/div/h5')
+    results = VersionPick({
+        Version.lowest(): Text(locator='//div[contains(@class, "toolbar-pf-results")]/div/h5'),
```

Comment:
REQUIRED: both xpaths are almost equal. Please use one widget with one common xpath instead.
You can always use 
*[self::i or self::img]
or something similar